lib/asset_checker.rb

```diff
@@ -0,0 +1,39 @@
+class AssetChecker
+  def self.run(argv)
+    assets_file = 'app/assets/javascripts/assets.js.erb'
+    translations_file = 'app/assets/javascripts/i18n-strings.js.erb'
+    @asset_strings = load_included_strings(assets_file)
+    @translation_strings = load_included_strings(translations_file)
+    exit_code = (argv.map { |f| check_file(f) }).any? ? 1 : 0
+    exit exit_code # rubocop:disable Rails/Exit
+  end
+
+  def self.check_file(file)
```

Comment:
Since this returns a boolean, can we rename this to be a predicate?

```ruby
def self.has_missing_assets_or_translations?(file)
```

I think it would improve readability for the return value check above too

Reply:
I like the idea, but the cruel power that is Rubocop says it's too long, so I'm changing to `missing_strings?` 👍